pycbc/waveform/compress.py

```diff
@@ -578,19 +622,21 @@ class CompressedWaveform(object):
         Writes the compressed waveform to an open hdf file.
     clear_cache :
         Clears the in-memory cache used to hold the
-        `sample_points`/`amplitude`/`phase`; only relevant if `load_to_memory`
-        is True.
+        `amplitude_freq_points`/`phase_freq_points`/`amplitude`/`phase`;
+        only relevant if `load_to_memory` is True.
 
     Class Methods
     -------------
     from_hdf :
         Loads a compressed waveform from the given open hdf file.
     """
-    
-    def __init__(self, sample_points, amplitude, phase,
+
+    def __init__(self,
+            amplitude_freq_points, phase_freq_points, amplitude, phase,
```

Comment:
As above, only a single `frequency_points` should be required, with an optional `amp_frequency_points` keyword argument set to None. Please also change the documentation string to reflect this.